blocklist-client/src/common/error.rs

```diff
@@ -3,10 +3,11 @@
 use reqwest::StatusCode;
 use serde::{Deserialize, Serialize};
 use utoipa::ToSchema;
-use warp::reject::Reject;
+use warp::{reject::Reject, reply::Reply};
 
 /// Errors occurring from Blocklist client's API calls to risk client and request handling
 #[derive(thiserror::Error, Debug)]
+#[allow(dead_code)]
```

Comment:
Is this still needed? Which code is dead?

Reply:
I like it for enums but I'll remove it.

Reply:
Removed.